instances/administrationType/intravenous.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/administrationType/intravenous",
```

Comment:
also here I would be more specific: `intravenousInjection` OR `intravenousInsertion` OR `intravenousAdministration`

Reply:
(cf. all comments/suggestions for intraperitoneal)